lib/tasks/database.rake

```diff
@@ -0,0 +1,74 @@
+require "mysql2"
+require "yaml"
+require "nokogiri"
+require "reverse_markdown"
+
+namespace :database do
+  namespace :legacy do
+    # NOTE to run remotely on heroku:
+    # Source: https://devcenter.heroku.com/articles/rake
+    #
+    # $ heroku run rake database:legacy:import_news
+    #
+    desc "Import news records to database"
+    task :import_news => :environment do
+      STDOUT.puts "This will destroy your news_items table. Enter 'Y' to confirm: [y/N]"
+      input = STDIN.gets.chomp
+      raise "Aborting import. You entered #{input}" unless input.downcase == "y"
+
+      NewsItem.destroy_all
+      records = YAML.load_file "#{Rails.root}/db/legacy.yml"
+      records.each do |data|
+        news_item = NewsItem.new
+        published_at = data["post_date"]
```

Comment:
`published_at` could be nil, if the original post is a draft (which will not happen with the collection I am parsing, as the original query generated only published posts with a date). So it's not really necessary to catch the non-existence of that key.

NOTE: It's a one-off task anyway